plugins/doc_fragments/ldap.py

```diff
@@ -24,6 +24,11 @@ class ModuleDocFragment(object):
       - The password to use with I(bind_dn).
     type: str
     default: ''
+  ca_cert_file:
```

Comment:
Sorry for not noticing this earlier; in ansible-core this parameter is called `ca_path` (https://docs.ansible.com/ansible/latest/collections/ansible/builtin/uri_module.html#parameter-ca_path, https://docs.ansible.com/ansible/latest/collections/ansible/builtin/url_lookup.html#parameter-ca_path). How about using that name here as well?